sql/hive-thriftserver/src/main/java/org/apache/hive/service/cli/operation/MetadataOperation.java

```diff
@@ -58,7 +55,7 @@ public void close() throws HiveSQLException {
   }
 
   /**
-   * Convert wildchars and escape sequence from JDBC format to datanucleous/regex
+   * Convert wildcards and escape sequence from JDBC format to datanucleus/regex
```

Comment:
I think this is from Hive. Let's just leave them.

Reply:
Okay.
I am still checking if there are other similar `typo` and submitting them together.